[2023-09-29T21:46:32.109+0200] {processor.py:157} INFO - Started process (PID=18680) to work on /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:46:32.110+0200] {processor.py:829} INFO - Processing file /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-29T21:46:32.110+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:46:32.110+0200] {dagbag.py:539} INFO - Filling up the DagBag from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:46:32.112+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:46:32.111+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-29T21:46:32.112+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:46:32.112+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-29T21:46:32.112+0200] {processor.py:841} WARNING - No viable dags retrieved from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:46:32.117+0200] {processor.py:179} INFO - Processing /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.009 seconds
[2023-09-29T21:47:18.863+0200] {processor.py:157} INFO - Started process (PID=18733) to work on /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:47:18.872+0200] {processor.py:829} INFO - Processing file /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-29T21:47:18.872+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:47:18.872+0200] {dagbag.py:539} INFO - Filling up the DagBag from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:47:18.874+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:47:18.874+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-29T21:47:18.874+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:47:18.874+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-29T21:47:18.874+0200] {processor.py:841} WARNING - No viable dags retrieved from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:47:18.879+0200] {processor.py:179} INFO - Processing /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2023-09-29T21:48:02.222+0200] {processor.py:157} INFO - Started process (PID=18922) to work on /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:48:02.231+0200] {processor.py:829} INFO - Processing file /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-29T21:48:02.232+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:48:02.231+0200] {dagbag.py:539} INFO - Filling up the DagBag from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:48:02.233+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:48:02.233+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-29T21:48:02.233+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:48:02.233+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-29T21:48:02.233+0200] {processor.py:841} WARNING - No viable dags retrieved from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:48:02.239+0200] {processor.py:179} INFO - Processing /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2023-09-29T21:48:43.461+0200] {processor.py:157} INFO - Started process (PID=19105) to work on /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:48:43.470+0200] {processor.py:829} INFO - Processing file /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-29T21:48:43.470+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:48:43.470+0200] {dagbag.py:539} INFO - Filling up the DagBag from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:48:43.472+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:48:43.471+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-29T21:48:43.472+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:48:43.472+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-29T21:48:43.472+0200] {processor.py:841} WARNING - No viable dags retrieved from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:48:43.477+0200] {processor.py:179} INFO - Processing /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2023-09-29T21:49:24.542+0200] {processor.py:157} INFO - Started process (PID=19178) to work on /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:49:24.551+0200] {processor.py:829} INFO - Processing file /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-29T21:49:24.551+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:49:24.551+0200] {dagbag.py:539} INFO - Filling up the DagBag from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:49:24.553+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:49:24.553+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-29T21:49:24.553+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:49:24.553+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-29T21:49:24.553+0200] {processor.py:841} WARNING - No viable dags retrieved from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:49:24.558+0200] {processor.py:179} INFO - Processing /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2023-09-29T21:50:05.966+0200] {processor.py:157} INFO - Started process (PID=19234) to work on /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:50:05.974+0200] {processor.py:829} INFO - Processing file /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-29T21:50:05.975+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:50:05.975+0200] {dagbag.py:539} INFO - Filling up the DagBag from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:50:05.976+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:50:05.976+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-29T21:50:05.976+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:50:05.976+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-29T21:50:05.977+0200] {processor.py:841} WARNING - No viable dags retrieved from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:50:05.981+0200] {processor.py:179} INFO - Processing /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2023-09-29T21:50:49.500+0200] {processor.py:157} INFO - Started process (PID=19290) to work on /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:50:49.501+0200] {processor.py:829} INFO - Processing file /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-29T21:50:49.501+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:50:49.501+0200] {dagbag.py:539} INFO - Filling up the DagBag from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:50:49.503+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:50:49.502+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-29T21:50:49.503+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:50:49.503+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-29T21:50:49.503+0200] {processor.py:841} WARNING - No viable dags retrieved from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:50:49.508+0200] {processor.py:179} INFO - Processing /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.010 seconds
[2023-09-29T21:51:31.849+0200] {processor.py:157} INFO - Started process (PID=19339) to work on /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:51:31.850+0200] {processor.py:829} INFO - Processing file /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-29T21:51:31.850+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:51:31.850+0200] {dagbag.py:539} INFO - Filling up the DagBag from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:51:31.852+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:51:31.851+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-29T21:51:31.852+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:51:31.852+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-29T21:51:31.852+0200] {processor.py:841} WARNING - No viable dags retrieved from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:51:31.857+0200] {processor.py:179} INFO - Processing /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.009 seconds
[2023-09-29T21:52:13.266+0200] {processor.py:157} INFO - Started process (PID=19403) to work on /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:52:13.274+0200] {processor.py:829} INFO - Processing file /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-29T21:52:13.274+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:52:13.274+0200] {dagbag.py:539} INFO - Filling up the DagBag from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:52:13.276+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:52:13.276+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-29T21:52:13.276+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:52:13.276+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-29T21:52:13.276+0200] {processor.py:841} WARNING - No viable dags retrieved from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:52:13.281+0200] {processor.py:179} INFO - Processing /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2023-09-29T21:52:55.714+0200] {processor.py:157} INFO - Started process (PID=19451) to work on /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:52:55.715+0200] {processor.py:829} INFO - Processing file /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-29T21:52:55.715+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:52:55.715+0200] {dagbag.py:539} INFO - Filling up the DagBag from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:52:55.717+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:52:55.716+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-29T21:52:55.717+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:52:55.717+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-29T21:52:55.717+0200] {processor.py:841} WARNING - No viable dags retrieved from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:52:55.722+0200] {processor.py:179} INFO - Processing /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.010 seconds
[2023-09-29T21:53:36.258+0200] {processor.py:157} INFO - Started process (PID=19511) to work on /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:53:36.266+0200] {processor.py:829} INFO - Processing file /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-29T21:53:36.266+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:53:36.266+0200] {dagbag.py:539} INFO - Filling up the DagBag from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:53:36.268+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:53:36.268+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-29T21:53:36.268+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:53:36.268+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-29T21:53:36.268+0200] {processor.py:841} WARNING - No viable dags retrieved from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:53:36.273+0200] {processor.py:179} INFO - Processing /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2023-09-29T21:54:38.110+0200] {processor.py:157} INFO - Started process (PID=19601) to work on /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:54:38.118+0200] {processor.py:829} INFO - Processing file /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-29T21:54:38.119+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:54:38.118+0200] {dagbag.py:539} INFO - Filling up the DagBag from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:54:38.120+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:54:38.120+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-29T21:54:38.120+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:54:38.120+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-29T21:54:38.120+0200] {processor.py:841} WARNING - No viable dags retrieved from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:54:38.126+0200] {processor.py:179} INFO - Processing /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2023-09-29T21:55:20.382+0200] {processor.py:157} INFO - Started process (PID=19756) to work on /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:55:20.390+0200] {processor.py:829} INFO - Processing file /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-29T21:55:20.391+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:55:20.391+0200] {dagbag.py:539} INFO - Filling up the DagBag from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:55:20.392+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:55:20.392+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-29T21:55:20.393+0200] {logging_mixin.py:151} INFO - [2023-09-29T21:55:20.392+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-29T21:55:20.393+0200] {processor.py:841} WARNING - No viable dags retrieved from /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-29T21:55:20.398+0200] {processor.py:179} INFO - Processing /home/radtrz/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
